src/dateutils.ts

```diff
@@ -57,6 +57,22 @@ export function month(date: XDate) {
 
   return fromTo(firstDay, lastDay);
 }
+/**
+ * Return a range of days from provided startDate that extends in the future for the 
+ * number of weeks provided by numberOfWeeksFromCurrent.
+ * 
+ * @param startDate Date to start add numberOfWeeksFromCurrent to
+ * @param numberOfWeeksFromCurrent Number of weeks which should be added to provided startDate
+ * @returns 
+ */
+function getDaysFromWeeksFromCurrent(startDate: XDate, numberOfWeeksFromCurrent: number) {
```

Comment:
The name now is very confusing. 
How about 'weeksDays(date: XDate, numberOfWeeks: number)`; to keep the naming similar to the other methods in this util? (Notice that your param name doesn't have to be equal to the prop name you pass to it)

Reply:
Will fix